### ISSUE TITLE
Design of loss function

### ISSUE THREAD

**Issue by Stevenisawesome520** [Nexuslkl/Swin_MIL#2]
Thank you for taking the time to read this.
I have a question about your paper Transformer based multiple instance learning for weakly supervised histopathology image segmentation.
I can't get the idea why the loss function works for this weakly supervised learning.
I don't know the reason why you take Generalized Mean value of every side-output and then compute the cross entropy using image-level label. I don't know why this works for weakly supervised learning for image segmentation.
I would appreciate it so much if you could reply me.
